servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -361,6 +394,13 @@ private FileInfo getFileInfoInternal(Inode inode) throws FileDoesNotExistExcepti
   }
 
   /**
+   * @return a read-only view of the inode tree
+   */
+  public FileSystemMasterView getFileStoreView() {
```

Comment:
Was renaming this to getFileSystemView() not considered? Is the difference between "FileStore" and "FileSystem" confusing?